django/db/models/functions/__init__.py

```diff
@@ -1,7 +1,13 @@
 """
```

Comment:
I think ideally this file would be `base.py` and `__init__.py` would be modeled after https://github.com/django/django/blob/353d436e7cb33cb832a3e8c74b051e3d2ba76018/django/core/files/__init__.py